src/net/sourceforge/kolmafia/Modifiers.java

```diff
@@ -2675,19 +2675,20 @@ public final void applyFloristModifiers() {
   public void applySynergies() {
     int synergetic = this.getRawBitmap(Modifiers.SYNERGETIC);
     if (synergetic == 0) return; // nothing possible
-    Iterator i = Modifiers.synergies.iterator();
+    Iterator<Entry<String, Integer>> i = Modifiers.synergies.entrySet().iterator();
     while (i.hasNext()) {
-      String name = (String) i.next();
-      int mask = ((Integer) i.next()).intValue();
+      Entry<String, Integer> entry = i.next();
+      String name = entry.getKey();
+      int mask = entry.getValue().intValue();
       if ((synergetic & mask) == mask) {
         this.add(Modifiers.getModifiers("Synergy", name));
       }
     }
   }
 
-  // Returned iterator yields alternating names / bitmaps
-  public static Iterator getSynergies() {
-    return Modifiers.synergies.iterator();
+  // Returned iterator yields bitmaps keyed by names
```

Comment:
Could then make this just return the map

Reply:
This, as well as multiple other getters, return iterators to make sure the underlying map doesn't get mutated (as far as I can tell)

Reply:
Then we can just make it an unmodifiable map

Reply:
but we WANT it to be modifiable. Just not from outside Modifiers.java...

Reply:
Yeah so you make it private and then use collections.unmodifiablemap in the getter